.github/workflows/integration.yml

```diff
@@ -271,6 +271,26 @@ jobs:
       - name: build proposals tests
         run: yarn build
         working-directory: a3p-integration
+      - name: zip up all core proposals
+        run: |
+          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
```

Comment:
I'll admit- I asked chatgpt to help me write this script

Reply:
nuthing wrong with that!

nit: the newlines aren't helping readability